misc/build.func

```diff
@@ -501,6 +502,41 @@ install_script() {
   fi
 }
 
+check_container_ressources() {
+  local required_ram="$var_ram"  # Expected RAM in MB
+  local required_cpu="$var_cpu"  # Expected number of Cores
+  local app_name="${APP}"        # Name of App
+
+  # Check actual RAM & Cores
+  local current_ram
+  current_ram=$(free -m | awk '/^Mem:/{print $2}')
+  current_cpu=$(nproc) 
+
+  # Check whether the current RAM is less than the required RAM or the CPU cores are less than required
+  if [[ "$current_ram" -lt "$required_ram" ]] || [[ "$current_cpu" -lt "$required_cpu" ]]; then
+    echo -e "\n⚠️${HOLD} ${GN}Required: ${required_cpu} CPU, ${required_ram}MB RAM ${CL}| ${RD}Current: ${current_cpu} CPU, ${current_ram}MB RAM${CL}"
+    echo -e "${YWB}Please ensure that the ${app_name} LXC is configured with at least ${var_cpu} vCPU and ${var_ram} MB RAM for the build process.${CL}\n"
```

Comment:
Not sure if we should block the user here or have a prompt like the storage check? Since not every update function requires a build process

Reply:
I think the community can accept that, even an apt-get upgrade can take >300MB, but if the memory is at 4GB and 3.6 of it is already occupied, it will be tight 😅 in the end it is a letter (y) :smile: